src/legacy/ui/public/agg_response/hierarchical/build_hierarchical_data.js

```diff
@@ -30,7 +30,6 @@ export const buildHierarchicalData = (table, { metric, buckets = [] }) => {
     slices = [{
       name: metricColumn.name,
       size: table.rows[0][metricColumn.id],
-      aggConfig: metricColumn.aggConfig
```

Comment:
Since these are always `undefined` I went ahead and removed them here so we aren't passing `aggConfig: undefined` around everywhere.